suse-caasp-plugin.8

```diff
@@ -1,4 +1,4 @@
-.TH suse-caasp-plugin "8" "09 Jun 2017" "suse-caasp-plugin" "Support Utilities Manual"
+.TH suse-caasp-plugin "8" "09 Jun 2020" "suse-caasp-plugin" "Support Utilities Manual"
```

Comment:
Was it _really_ in the 9th of June that it was updated?  :)  If I'm being picky, I'd say you should adjust the whole date.

Reply:
I scrolled past this a dozen times and refrained from saying anything. Thanks for scratching that itch. 😂